### ISSUE TITLE
PostgreSQL 14.11 is not able to identify the postmaster daemon.

### ISSUE THREAD

**Issue by vigneshkumar2016** [docker-library/postgres#1208]
PostgreSQL 14.11 runs out on error.
Image: Official Image
Version: 14.11
OS: Debian
OS version: bullseye
Container: yes

PostgreSQL errors out on 14.11
Segmentation fault (core dumped). this is quite strange.

PostgreSQL 14.5, 14.6, 14.0 works just fine.

Seems like the compiler is overshooting on the address space to load postgres.

any help here is much appreciated.

**Comment by vigneshkumar2016** [comment]
> libseccomp

Thanks, this works for me. However the traceback results and coredump looks similar with that of the one which is working and failing. pretty strange to debug. But clearly getting libraries are not getting loaded to the container memory.

**Comment by tianon** [comment]
Yeah, the seccomp blocks happen at the kernel level, so strace is one of the few tools that shows what's happening and even then it's usually an obscure `EPERM` in places it shouldn't be and no real indicator that it's actually seccomp blocking it. :sob:
